addons/missileGuidanceCompat/patchGM/config.cpp

```diff
@@ -0,0 +1,157 @@
+#include "\z\potato\addons\missileGuidanceCompat\script_component.hpp"
+#undef COMPONENT
+#define COMPONENT missileGuidanceCompat_patchGM
+
+class CfgPatches {
+    class ADDON {
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "potato_missileGuidanceCompat",
+            "gm_weapons_ammo",
+            "gm_weapons_launchers_fagot",
+            "gm_weapons_launchers_hot",
+            "gm_weapons_launchers_maljutka",
+            "gm_weapons_launchers_milan"
+        };
+        skipWhenMissingDependencies = 1;
+        author = "Bourbon Warfare";
+        authorUrl = "https://github.com/BourbonWarfare/POTATO";
+        VERSION_CONFIG;
+    };
+};
+
+class CfgAmmo {
+    class gm_missile_saclos_base;
+    // Bastion
+    class gm_missile_bastion_base: gm_missile_saclos_base {
+        maneuvrability = 0;
+        #include "../CfgMissileBastion.hpp"
+    };
+    class gm_missile_bastion_heat_9M117: gm_missile_bastion_base {
+        class ace_missileguidance: ace_missileguidance {
+            enabled = 1;
+            pitchRate = 25;
+            yawRate = 25;
+            lineGainP = 6;
+            lineGainD = 5;
+            showTrail = 0;
```

Comment:
generally I enable `showTrail` for player guided missiles to see them at a distance, not blocking though

Reply:
GM adds some stuff to their models to make them fairly visible, I'll post pictures after work.

Reply:
yeah im good with it in this case then, ill look at the images but i trust it's decent

Reply:
on discord in scripting